dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -828,11 +836,11 @@
 			clocks = <&ccm IMX_CCM_SAI2_CLK 0x7C 2>;
 			pre-div = <0>;
 			podf = <63>;
-			pll-clocks = <&anatop 0x70 0xC000 0>,
-				   <&anatop 0x70 0x7F 32>,
-				   <&anatop 0x70 0x18 1>,
-				   <&anatop 0x80 0x3FFFFFFF 77>,
-				   <&anatop 0x90 0x3FFFFFFF 100>;
+			pll-clocks = <&anatop 0x70 0xC000 0x0>,
+					<&anatop 0x70 0x7F 32>,
+					<&anatop 0x70 0x180000 1>,
+					<&anatop 0x80 0x3FFFFFFF 0>,
```

Comment:
Any reason the numerator for this PLL is 0 and not 77?

Reply:
Reverted--this was for some internal testing with a different MCLK value.